core/templates/dev/head/css/oppia.css

```diff
@@ -4558,3 +4558,9 @@ md-card.preview-conversation-skin-supplemental-card {
   padding: 0;
   width: 1px;
 }
+
+
+.popover.bottom.fade.in {
```

Comment:
For example. see e.g. https://github.com/oppia/oppia/pull/3409/files#diff-a5de01b7be4948894c7b8f8782677062R4535 where the selector is only applied to .diff-graph elements within .oppia-save-exploration-wide-modal and not .diff-graph elements throughout the entire site.